### PR TITLE
feat: created karate assignment

### PR DESCRIPTION
**Pull request by harsha619**
karate assignment :
it continas the testing of api using karate

### CODE REVIEW

File: karate-assignment/src/test/java/todoist/comments/comment-by-id.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@commentById
```
Comment (samhithav27) [inline karate-assignment/src/test/java/todoist/comments/comment-by-id.feature]: rename file to 'get-comment-by-id'

File: karate-assignment/src/test/java/todoist/comments/comment-list.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@allComments
```
Comment (samhithav27) [inline karate-assignment/src/test/java/todoist/comments/comment-list.feature]: double on all naming conventions

File: karate-assignment/src/test/java/todoist/comments/create-comment.feature [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  Scenario: Add new comment to todoist
+    Given path 'comments'
+    And header Authorization = 'Bearer ' + ACCESS_TOKEN
+    And header Content-Type = 'application/json'
+    * def content = "Need one bottle of milk"
+
+    And request
+      """
+      {
```
Comment (samhithav27) [inline karate-assignment/src/test/java/todoist/comments/create-comment.feature:17]: move this to a fixtures file

File: karate-assignment/src/test/java/todoist/labels/delete-label.feature [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  Scenario: Delete the label by id
+    Given path 'labels/' + labelId
+    And header Authorization = 'Bearer ' + ACCESS_TOKEN
+    And header Content-Type = 'application/json'
+    When method delete
+    Then status 204
+
+
+
+
```
Comment (samhithav27) [inline karate-assignment/src/test/java/todoist/labels/delete-label.feature:16]: add more edge cases and test cases
